data/campaigns/Heir_To_The_Throne/units/Battle_Princess.cfg

```diff
@@ -120,7 +122,10 @@
             {ABILITY_INITIATIVE}
         [/abilities]
         usage=fighter
-        description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}+{SPECIAL_NOTES_INITIATIVE}+{SPECIAL_NOTES_MAGICAL}
+        description= _ "A noble by birth, the Princess has learnt swordplay with the greatest generals and battle tactics with the greatest sages, making her both a great combatant and a leader. Now battle-hardened and strong of will, she can aid those around her in the art of combat."{SPECIAL_NOTES}
```

Comment:
Leftover `{SPECIAL_NOTES}` here, and in Commander.cfg in the same directory. Didn't check if there were other cases.